app/components/tab.vue

```diff
@@ -1,9 +1,56 @@
 <template>
-  <p>Hello!</p>
+  <div>
+    <p>FotoFluent</p>
+    <Translation :translation="translation" v-if="translation"></Translation>
+  </div>
 </template>
 
 <script>
+  import axios from 'axios'
+  import _ from 'lodash'
+  import Translation from './translation.vue'
+
   export default {
-    name: 'tab'
+    name: 'tab',
+    components: {
+      Translation
+    },
+    data() {
+      return {
+        translation: null,
+        image: null      }
+    },
+    mounted() {
+      this.requestData()
+    },
+    methods: {
+      requestData() {
+        axios.get(`https://fotofluent-admin.herokuapp.com/translations.json`)
+          .then(response => {
+            this.translation = _.sample(response.data)
+            this.image = this.translation.word.image
+            document.body.style.backgroundImage = `url(${this.image})`;
+          })
+          .catch(err => {
+            console.log(err)
+          })
+      }
+    }
   }
 </script>
+
+<style>
+  body {
+    background: url('https://images.unsplash.com/photo-1482100199117-a4a38a64e7e3?dpr=1') no-repeat center center fixed;
```

Comment:
So right now that's the default image that gets added while the API response comes in and then it's replaced by the dynamic image associated with the word. We'll probably change it later, maybe so it's some sort of loading page with the FF logo till the response is ready.